babel/src/main/codegen/config.fmpp

```diff
@@ -857,6 +858,16 @@ data: {
       dropStatementParserMethods: [
       ]
 
+      # Binary operators tokens
+      binaryOperatorsTokens: [
+        {name: "PG_CAST", symbol: "::"}
+      ]
+
+      # Binary operators initialization
+      extraExpressions: [
+        "op = PostgreSQLCasting(list, exprContext)"
+      ]
```

Comment:
Should we need to extend this ? The keyword "name", "symbol" or "exprContext" seems too context relevant. Such keywords should not appears in this config template.

Reply:
What do you suggest instead ?

Reply:
These parse block hooks are not necessary, if you want to add a new sql function parse, extend the expression parse block. See https://github.com/apache/calcite/commit/7f33215ffaf9c0b8f4bef082913c910c77bf4427 for reference.